cms/views_test.py

```diff
@@ -580,3 +581,18 @@ def test_blog_page_context(client, wagtail_basics):
     context = resp.context_data
 
     assert "posts" in context
+
+
+def test_enterprise_page_context(client, wagtail_basics):
+    """
+    Test that enterprise page show correctly
+    """
+    enterprise_page = EnterprisePageFactory.create(
+        parent=wagtail_basics.root, action_title="Read More", description="description"
+    )
+    enterprise_page.save_revision().publish()
+
+    resp = client.get(enterprise_page.get_url())
+
+    assert resp.status_code == status.HTTP_200_OK
+    assert resp.context_data["page"] == enterprise_page
```

Comment:
How about we check for all the other context data to be present in the context?

Reply:
We can skip the creation of subpages but we should ensure that the keys are there in the context.